src/data/bucket/symbol_bucket.js

```diff
@@ -413,7 +414,9 @@ class SymbolBucket implements Bucket {
                 if (formattedText.containsRTLText()) {
                     this.hasRTLText = true;
                 }
-                text = transformText(formattedText, layer, feature);
+                if (this.hasRTLText && globalRTLTextPlugin.isLoaded() || !this.hasRTLText) {
+                    text = transformText(formattedText, layer, feature);
```

Comment:
Should there be a warning emitted here so that customers don't just see a map with no labels and wonder what the heck is going on?

Reply:
I've changed the behavior now so that only when lazy loading, does it skip rendering the text.